cmd/cleantool/cleantool.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v52/github"
+	"github.com/sethvargo/go-githubactions"
+	"golang.org/x/oauth2"
+)
+
+func main() {
```

Comment:
Could you separate the logic related to getting stale versions in a separate function? Then that function can return the slice of stale version IDs :smile:

Reply:
added a function to check whether the package need to be clean